app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -111,6 +111,24 @@ describe("the new project page", () => {
     cy.happoAndAxe("Project summary Form", "empty", "main", true);
   });
 
+  it("renders the IA specific project forms", () => {
+    cy.mockLogin("cif_admin");
+    cy.visit("/cif/projects");
+    cy.get("button").contains("Add a Project").click();
+
+    // NEW
+    cy.url().should("include", "/new");
+    cy.findByLabelText(/Funding Stream$/i).should("be.visible");
+    cy.happoAndAxe("IA Project New Form", "unfilled", "main");
```

Comment:
I don't know that we need this a screenshot of this--it'll look the same as the empty EP form